### PR TITLE
[4.0] Add missing Table Caption

### PR DESCRIPTION
**Pull request by hans2103**
Pull Request for Issue https://github.com/joomla/joomla-cms/pull/30761#issuecomment-698776641 .

### Summary of Changes

This PR adds the HTML for a missing Table Caption

### Testing Instructions

Joomla 4 with "Testing Sample Data"
url =>index.php/article-category-list

### Actual result BEFORE applying this Pull Request

Inspect element on shown table.
No table caption is visible

Schermafbeelding 2020-09-25 om 11 23 14

### Expected result AFTER applying this Pull Request

Table caption is visible
Schermafbeelding 2020-09-25 om 11 27 04

### CODE REVIEW

**Comment by infograf768** [comment]
String is necessary in frontend
COM_CONTENT_ARTICLES_TABLE_CAPTION="Table of Articles"
Screen Shot 2020-09-25 at 11 54 43

File: components/com_content/tmpl/category/default_articles.php [hunk 1]
```diff
@@ -117,6 +117,9 @@
 <?php else : ?>
 
 	<table class="com-content-category__table category table table-striped table-bordered table-hover">
+		<caption id="captionTable" class="sr-only">
+			<?php echo Text::_('COM_CONTENT_ARTICLES_TABLE_CAPTION'); ?>,
```
Comment (brianteeman) [inline components/com_content/tmpl/category/default_articles.php]: Remove the extra ,

```suggestion
			<?php echo Text::_('COM_CONTENT_ARTICLES_TABLE_CAPTION'); ?>
```

**Comment by brianteeman** [comment]
I think you forgot to commit the updated language file

**Comment by hans2103** [comment]
> I think you forgot to commit the updated language file

now I did... I found a language string. Forgot to see that it was in administrator/language, not in language
Now it is

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on ca80afa7c83d11582ccceddaf342243ec5012a23<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30763">issues.joomla.org/tracker/joomla-cms/30763</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ca80afa7c83d11582ccceddaf342243ec5012a23<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30763">issues.joomla.org/tracker/joomla-cms/30763</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30763">issues.joomla.org/tracker/joomla-cms/30763</a>.</sub>

**Comment by infograf768** [comment]
Tks.

**Pull request merged**